worker/index.js

```diff
@@ -63,6 +64,9 @@ async function work () {
   await boss.work('views', views(args))
   await boss.work('rankViews', rankViews(args))
 
+  // Run once
```

Comment:
I added this to the worker because I didn't see a nice way to add it on start-up in the main `app` container. Also, this isn't really critical functionality, so I figured it made sense to put it in the separate worker process anyway. Open to feedback on this.

Reply:
Was thinking more about adding a line to `.platform/hooks/predeploy/00_build.sh` but I guess is fine here, too

Probably also easier to do it here since you already have database access

Reply:
> Was thinking more about adding a line to .platform/hooks/predeploy/00_build.sh but I guess is fine here, too

So add a new `npm script`, or maybe just a script in the `scripts` folder, then e.g. `node scripts/seedContributors.js` in the build.sh? That would also be fine with me. I wasn't sure the best place, so I just stuck it here.

Reply:
Nah, I think it's fine / better here because of no need to configure database access.